src/app/auth/auth.model.ts

```diff
@@ -14,7 +15,13 @@ export interface ActionHandlerList {
   thirdStep: VoidCallback;
 }
 
+type NestedFormInterface<T, N> = {
+  [K in keyof T]: FormControl<N>;
```

Comment:
```suggestion
  [key in keyof T]: FormControl<N>;
```
Capital case looks like generic, it is very confused

Reply:
fixed